be/src/olap/hll.h

```diff
@@ -21,6 +21,7 @@
 #include <math.h>
 #include <stdio.h>
 
+#include <string>
```

Comment:
pay attention to the order of include,
string should under set

Reply:
fixed